packages/http-client/src/http-client.ts

```diff
@@ -445,6 +451,7 @@ async function getCsrfHeaders(
         params: request.params,
         headers: request.headers,
         url: request.url,
+        timeout: request.timeout,
```

Comment:
```suggestion
        timeout: request.timeout || defaultTimeoutTarget,
```

The last commit (which changed this) seems to have broken a test.